eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -69,6 +102,25 @@ jobs:
           ${{ insert }}: ${{ parameters.TestStepMavenInputs }}
         env: ${{ parameters.EnvVars }}
 
+      - ${{ parameters.PostRunSteps }}
+
+      - pwsh: >-
+          eng/Remove-TestResources.ps1
+          -BaseName $(BaseName)
+          -ServiceDirectory ${{ parameters.ServiceDirectory }}
+          -TenantId $env:TENANT_ID
+          -ProvisionerApplicationId $env:PROVISIONER_APPLICATION_ID
+          -ProvisionerApplicationSecret $env:PROVISIONER_APPLICATION_SECRET
+          -Force
+          -Verbose
+        env:
+          TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+          PROVISIONER_APPLICATION_ID: $(provisioner-aad-id)
+          PROVISIONER_APPLICATION_SECRET: $(provisioner-aad-secret)
+        displayName: Remove Test Resources
+        condition: true
```

Comment:
@danieljurek: Should this be `condition: always()` instead?  Or are these equivalent?  I assume the intent is to ensure cleanup is always run regardless of earlier failures.